### PR TITLE
Minor improvement in FIZZBUZZTEST

### PR DESCRIPTION
**Pull request by f-lombardo**

### CODE REVIEW

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
-        this.bif_found() != null -> this.bif_found().toAst(conf)
-        this.bif_eof() != null -> this.bif_eof().toAst(conf)
-        this.bif_abs() != null -> this.bif_abs().toAst(conf)
-        this.bif_int() != null -> this.bif_int().toAst(conf)
-        this.bif_editw() != null -> this.bif_editw().toAst(conf)
-        this.bif_rem() != null -> this.bif_rem().toAst(conf)
-        this.bif_replace() != null -> this.bif_replace().toAst(conf)
-        else -> TODO(this.text + " " + toPosition(conf.considerPosition))
+    val position = toPosition(conf.considerPosition)
+    return enrichPossibleExceptionWith("Error at line ${position?.line()} - ${this.text}") {
```
Comment (ftomassetti) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt]: enrichPossibleExceptionWith seems a nice idea to me!

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
+            this.bif_diff() != null -> this.bif_diff().toAst(conf)
+            this.bif_editc() != null -> this.bif_editc().toAst(conf)
+            this.bif_found() != null -> this.bif_found().toAst(conf)
+            this.bif_eof() != null -> this.bif_eof().toAst(conf)
+            this.bif_abs() != null -> this.bif_abs().toAst(conf)
+            this.bif_int() != null -> this.bif_int().toAst(conf)
+            this.bif_editw() != null -> this.bif_editw().toAst(conf)
+            this.bif_rem() != null -> this.bif_rem().toAst(conf)
+            this.bif_replace() != null -> this.bif_replace().toAst(conf)
+            else -> TODO()
```
Comment (ftomassetti) [inline rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt]: This message is less expressive than before

File: rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/parsing/RpgParserWithMuteRuntimeTest.kt [hunk 3]
```diff
@@ -139,7 +139,6 @@ class RpgParserWithMuteRuntimeTest {
 
     @Test
     fun executingFIZZBUZZTEST() {
-        DummySystemInterface.executedAnnotationInternal.clear()
```
Comment (ftomassetti) [inline rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/parsing/RpgParserWithMuteRuntimeTest.kt]: This helped with fuzzy tests before. Is this not needed anymore?

Comment (f-lombardo) [reply rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/parsing/RpgParserWithMuteRuntimeTest.kt]: We don't use the DummySystemInterface here, but the SimpleSystemInterface, so I think that the cleaning is useless here, isn't it?

File: rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt (hunk 2 above)
Comment (f-lombardo) [reply rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt]: Well, I think that the description of the not implemented bif is already given by the enrichPossibleExceptionWith method

**Pull request merged**